Service/UpdateChecklistService.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+namespace CommonGateway\HuwelijksplannerBundle\Service;
+
+use App\Entity\ObjectEntity;
+use App\Service\ObjectEntityService;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use function Symfony\Component\DependencyInjection\Loader\Configurator\param;
+
+/**
+ * This service holds al the logic for checksing data from the marriage request and updating the associated checklist.
+ */
+class UpdateChecklistService
+{
+    private EntityManagerInterface $entityManager;
+    private ObjectEntityService $objectEntityService;
```

Comment:
You should never use this objectEntityService service in this file!
Because this service is old and depricated